lib/index.js

```diff
@@ -23,17 +23,25 @@ const rootDir = __dirname.endsWith("dist/lib")
     : resolve(__dirname, "..");
 const bindingsPath = getBindingsPath(rootDir);
 
+function convertToHex(array) {
+  // padd with leading 0 if <16
+  function i2hex(i) {
+    return ("0" + i.toString(16)).slice(-2);
+  }
+  return array.reduce((ret, i) => ret + i2hex(i), "");
+}
+
 function prepareBindings(bindings) {
   bindings.SecretKey.prototype.toHex = function toHex() {
-    return `0x${this.serialize().toString("hex")}`;
+    return `0x${convertToHex(this.serialize())}`;
```

Comment:
I would assume this is a bit slower? Maybe could just do
```suggestion
    return `0x${Buffer.from(this.serialize().buffer).toString("hex")}`;
```

Reply:
You are correct!!! 🚀 

```sh

  utils
    ✓ convertToHex                                                        615763.5 ops/s    1.624000 us/op        -     346693 runs  0.606 s
    ✓ Buffer.from(testKey.buffer).toString                                 4273504 ops/s    234.0000 ns/op        -    1507263 runs  0.505 s
```